modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/GcalEpics.scala

```diff
@@ -0,0 +1,138 @@
+package edu.gemini.seqexec.server
+
+import java.util.logging.Logger
+
+import edu.gemini.epics.acm.{CaAttribute, CaService, CaStatusAcceptor, XMLBuilder}
+import edu.gemini.seqexec.server.gcal.BinaryOnOff
+
+/**
+  * Created by jluhrs on 3/14/17.
+  */
+class GcalEpics(epicsService: CaService) {
+
+  import EpicsCommand.setParameter
+  import GcalEpics._
+
+  object shutterCmd extends EpicsCommand {
+    override val cs = Option(epicsService.getCommandSender("gcal::shutter"))
```

Comment:
That is not an EPICS channel name, but the name assigned to a Command Sender. Because all the Command Senders are in the same space, the name use that convention to avoid conflicts.